tests/unit/test-class-wc-payments-order-service.php

```diff
@@ -697,14 +697,14 @@ public function test_mark_payment_capture_failed_null_intent_status() {
 	public function test_mark_payment_capture_expired_with_provider( $fraud_outcome, $expected_fraud_outcome, $expected_fraud_meta_box ) {
 		// Arrange: Create the intent, get the proper order status variations, and get the charge. Set the fraud outcome status.
 		$intent            = WC_Helper_Intention::create_intention( [ 'status' => Payment_Intent_Status::CANCELED ] ); // Stripe uses single 'l'.
```

Comment:
The comment seems to be not relevant anymore, or that's just me?

Reply:
No, it's me :) . Thanks for the callout.  Notes to myself - Watch the comments around the code thats's changed!

Reply:
Thanks for spotting. I have removed it.